packages/react/src/components/NumberInput/NumberInput-story.js

```diff
@@ -80,16 +80,7 @@ export default {
 };
 
 export const Default = () => {
-  return (
-    <NumberInput
-      min={0}
-      max={100}
-      value={50}
-      label="NumberInput label"
-      helperText="Optional helper text."
-      invalidText="Number is not valid"
-    />
-  );
+  return <NumberInput {...props()} />;
```

Comment:
Can we keep this as-is so that the default story can be copy / pasted? We can add in additional props to the component if we want to display them, but we're trying to only use the `...props()` syntax in the `Playground` example